pages/code-of-conduct.tsx

```diff
@@ -45,97 +46,130 @@ export default withPageMetadata((props: WithPageMetadataProps) => (
 
     <h2>Need Help?</h2>
 
-    <p>
+    <StyledPara>
       If someone makes you or anyone else feel unsafe or unwelcome, please report it as soon as possible. Conference
-      staff can be identified by {props.pageMetadata.conference.Organiser.ShirtColour} event branded t-shirts and
-      organiser name tags. Harassment and other Code of Conduct violations reduce the value of our event for everyone.
-      We want you to be happy at our event. People like you make our event a better place. You can make a report either
-      personally or anonymously.
-    </p>
+      staff can be identified by {pageMetadata.conference.Organiser.ShirtColour} event branded t-shirts and organiser
+      name tags. Harassment and other Code of Conduct violations reduce the value of our event for everyone. We want you
+      to be happy at our event. People like you make our event a better place. You can make a report either personally
+      or anonymously.
+    </StyledPara>
 
     <h3>Personal report</h3>
 
-    <p>You can make a personal report by:</p>
+    <StyledPara>You can make a personal report by:</StyledPara>
 
     <StyledList>
       <li>
-        Contacting a staff member, identified by {props.pageMetadata.conference.Organiser.ShirtColour} event branded
-        t-shirts and organiser name tags
+        Contacting a staff member, identified by {pageMetadata.conference.Organiser.ShirtColour} event branded t-shirts
+        and organiser name tags
```

Comment:
is saying colour and type of badge the best option here? Perhaps we should give more detail, more options and a photo. Shouldn't rely solely on colour for a description.

Reply:
special lanyards? wearing a cap?

Reply:
could we raise it as an issue on your planner board to be discussed in our next committee meeting?